stories/NewSelectField/NewSelectField.stories.js

```diff
@@ -39,7 +39,7 @@ storiesOf('new select field', module)
     'object select field',
     () => {
       return {
-        template: hbs`{{new-bourbon-select-field content=petObject optionLabelPath="content.label" optionValuePath="content.value" fullWidth=true}}`,
+        template: hbs`{{new-bourbon-select-field content=petObject optionLabelPath="content.label" optionValuePath="content.value" fullWidth=true value='the other Brussels Griffon'}}`,
```

Comment:
Is there a reason these `value`s need to be added? The behavior regarding setting default values shouldn't have changed, right?

Reply:
it didn't change.  just realized i didn't have an example for this case so just added it.